tests/snapshots/assets__check_file@cetz-manual.typ-40.snap

```diff
@@ -254,15 +258,13 @@ Marks are arrow tips that can be added to the end of path based elements that su
     columns: 3,
     [*Name*], [*Shorthand*], [*Shape*],
     ..(
-      for (
-        name,
-        item,
-      ) in cetz.mark-shapes.marks {
+      for (name, item) in cetz
+        .mark-shapes
+        .marks {
         let name-to-mnemonic = (:)
-        for (
-          name,
-          item,
-        ) in cetz.mark-shapes.mnemonics {
+        for (name, item) in cetz
+          .mark-shapes
+          .mnemonics {
```

Comment:
maybe we should only enable this for chains that contains function call? I'm not sure..